Process.yml

```
@@ -759,7 +759,11 @@ $graph:
   extends: [Parameter, InputFormat, LoadContents]
   fields:
     - name: default
-      type: Any?
+      type:
+        - "null"
+        - File
+        - Directory
+        - Any
```

I think this could be ..
```suggestion
        - CWLType
        - InputRecordSchema
        - InputArraySchema
        - Any
```

---

Ah, we don't have a `recordLoader` nor `arrayLoader` yet in Python codegen.. might be nice, though..

---

This is the "default value" field.  It contains actual values.  It isn't a "type" field.

The behavior I need to happen here is for the codegen to interpret File and Directory as meaningful object types so that "location" gets subject to URI expansion.

Having the type of "default" be "CWLType" means it'll accept the literal string "File", not a file object.